src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleZoot.kt

```diff
@@ -0,0 +1,40 @@
+package net.ccbluex.liquidbounce.features.module.modules.player
+
+import net.ccbluex.liquidbounce.event.repeatable
+import net.ccbluex.liquidbounce.features.module.Category
+import net.ccbluex.liquidbounce.features.module.Module
+import net.minecraft.network.packet.c2s.play.PlayerMoveC2SPacket
+
+object ModuleZoot : Module("Zoot", Category.PLAYER) {
+
+    val badEffects by boolean("BadEffects", true)
+    val fire by boolean("Fire", true)
+    val noAir by boolean("NoAir", false)
+
+    val repeatable = repeatable {
+        if (!noAir && player.isOnGround) {
+            if (badEffects) {
+                val effect = player.activeStatusEffects.maxByOrNull { it.value.duration }
+
+                if (effect != null) {
+                    if(effect.value.isPermanent) {
+                        player.statusEffects.remove(effect.value)
```

Comment:
Are you crazy?

Reply:
Don't just remove the potion effects from the client side.

Reply:
Please fix this.

Reply:
How should I remove potions server side ? Mixins again?

Reply:
Found a packet for that.